Remotion/Web/Development.WebTesting/HostingStrategies/AspNetDockerContainerWrapper.cs

```diff
@@ -25,16 +26,17 @@ public AspNetDockerContainerWrapper (IDockerClient docker, AspNetDockerContainer
     }
 
     _processPath = configurationParameters.ProcessPath;
+    _processArguments = configurationParameters.ProcessArguments;
   }
 
   protected override string GetEntryPoint ()
   {
-    return Path.GetFullPath(Path.Combine(ConfigurationParameters.AbsoluteWebApplicationPath, _processPath));
+    return _processPath;
```

Comment:
This is because of  TestSiteLayoutConfiguration, correct?

Reply:
This was always a bit sketchy because `_processPath` is already rooted. As such, `Path.Combine(left, right)` gets to absolute paths and thus returns the `right` one ;)

To allow invocations like `dotnet` a relative path should not be made absolute here.

Reply:
should we add a unit test with a well-named test value (e.g. "dotnet") or at least a comment in the code? It feels like this is something someone might easily "fix" accidentally.